src/main.js

```diff
@@ -5,12 +5,16 @@ import './libs/actions'
 import App from './panel/app_panel'
 import PanelManager from "./proxies/panel_manager";
 import UrlState from "./proxies/url_state";
+import Store from "./adapters/store";
 
 
 (async function main() {
   new I18n()
   await setLang()
 
+  const store = new Store()
+  await store.onConnect()
```

Comment:
are we sure we want to freeze app start-up to get store data ?

Reply:
changed to start execute onConnect at instantiation and await the resolution of the promise only when needed (at method execution)